src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +35,16 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+
+        AtomicInteger index = new AtomicInteger();
+
+        return Arrays.stream(numbers.toArray())
```

Comment:
We can use IntStream.range(0, numbers.size()) instead of creating new variable and reduce code only with 4 methods (map, filter, average, orElseThrow)
```suggestion
        return IntStream.range(0, numbers.size())
                                   ...
```

Reply:
I was not successful with this. How to iterate? left my version